tests/sampling/test_mcmc.py

```diff
@@ -507,6 +507,14 @@ def test_empty_model():
         error.match("any free variables")
 
 
+def test_blas_cores():
+    with pm.Model():
+        pm.Normal("a")
+        pm.sample(blas_cores="auto", tune=10, cores=2, draws=10)
+        pm.sample(blas_cores=None, tune=10, cores=2, draws=10)
+        pm.sample(blas_cores=2, tune=10, cores=2, draws=10)
```

Comment:
Can we actually test anything? I guess not in a reasonable amount of logic

Reply:
I couldn't think of any way to check if it really does what it's supposed to do.
I checked locally that it uses the correct number of threads, but otherwise I don't really know anything.

Reply:
Fair enough